vars/buildWDKAutoSwitch.groovy

```diff
@@ -10,22 +10,27 @@ import net.wooga.jenkins.pipeline.BuildUtils
 //                                                                                                                    //
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-def call(Map config = [unityVersions:[]]) {
+def call(Map config = [ unityVersions:[] ]) {
 
   def buildUtils = new BuildUtils()
-  def unityVersions = buildUtils.parseVersions(config.unityVersions)
+
+  // We need at least one valid unity version for now.
+  config.unityVersions = buildUtils.parseVersions(config.unityVersions)
+  if(config.unityVersions.isEmpty()) {
+    error "Please provide at least one unity version."
+  }
+
+  // Set defaults as needed
   config.testEnvironment = config.testEnvironment ?: []
   config.testLabels = config.testLabels ?: []
   config.labels = config.labels ?: ''
-  //we need at least one valid unity version for now.
-  if(unityVersions.isEmpty()) {
-    error "Please provide at least one unity version."
-  }
+  config.refreshDependencies = config.refreshDependencies ?: false
+  config.logLevel = config.logLevel.toString() ?: "info" // quiet, warn, info, debug
```

Comment:
I would like to see these two values as runtime paramter which are selected when starting the build.

Reply:
You got it! (I wanted to do that as well)